### ISSUE TITLE
Listing Purpose

### ISSUE THREAD

**Issue by chf2** [edhuang24/ShireBnB#2]
I'm not 100% sure I understand why we have listing tables separate from houses. Are houses not always listed? If that's so, does it make more sense for the price to be associated with the listing than with the house?

**Comment by edhuang24** [comment]
yeah, i was struggling with figuring out best practice over the weekend. i thought that sometimes owners might not want their house listed at all times, in which case there would be a "de-list" option, but i guess airbnb deploys this by blocking out dates during which the listing is not available? i think your suggestion is easier to implement and makes more sense, honestly. just wasn't sure what was best at first.
